android/java/org/chromium/chrome/browser/BraveHelper.java

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser;
+
+import org.chromium.base.CommandLine;
+import org.chromium.base.ContextUtils;
+import org.chromium.chrome.browser.ChromeSwitches;
+import org.chromium.chrome.browser.datareduction.DataReductionPromoUtils;
+import org.chromium.chrome.browser.firstrun.FirstRunStatus;
+
+public class BraveHelper {
+  public static final String SHARED_PREF_DISPLAYED_INFOBAR_PROMO =
+            "displayed_data_reduction_infobar_promo";
+
+  public BraveHelper() {}
+
+  public static void DisableFREDRP() {
+      // Disable FRE for arm64 builds where ChromeActivity is the one that
+      // triggers FRE instead of ChromeLauncherActivity on arm32 build.
```

Comment:
This comment was meant to explain why the same thing is written in BraveActivity while we already have it in BraveLauncherActivity, it seems a bit off to be directly moved here without modification.
Since you pull it into a helper function, how about we add some comments for this function above `public static void DisableFREDRP() {` instead to state this is meant to set some state during startup and it would be called by BraveActivity on arm64 and BraveLauncherActivity on arm32?

Reply:
Just saw the original comment was still kept in BraveActivity, then I think just remove these two lines is fine.

Reply:
Yep, deleted this 2